libs/libzauth/libzauth-c/src/zauth.h

```diff
@@ -49,14 +55,15 @@ void        zauth_keystore_delete(ZauthKeystore * store);
 ZauthResult zauth_acl_open(uint8_t const * fname, size_t len, ZauthAcl **);
 void        zauth_acl_delete(ZauthAcl * store);
 
-ZauthResult    zauth_token_parse(uint8_t const * str, size_t len, ZauthToken **);
-ZauthResult    zauth_token_verify(ZauthToken const *, ZauthKeystore const *);
-ZauthTokenType zauth_token_type(ZauthToken const *);
-long           zauth_token_time(ZauthToken const *);
-uint8_t        zauth_token_version(ZauthToken const *);
-Range          zauth_token_lookup(ZauthToken const *, uint8_t);
-ZauthResult    zauth_token_allowed(ZauthToken const *, ZauthAcl const *, uint8_t const * path, size_t len, uint8_t * result);
-void           zauth_token_delete(ZauthToken *);
+ZauthResult            zauth_token_parse(uint8_t const * str, size_t len, ZauthToken **);
+ZauthResult            zauth_token_verify(ZauthToken const *, ZauthKeystore const *);
+ZauthTokenType         zauth_token_type(ZauthToken const *);
+ZauthTokenVerification zauth_token_verification(ZauthToken const *);
```

Comment:
note to self: this is the only line that has been changed in this diff block (except for indentation).